sdstore/cache2disk.go

```diff
@@ -0,0 +1,166 @@
+package sdstore
+
+import (
+	"fmt"
+	"path/filepath"
+	"os"
+	"strings"
+	"io"
+	"io/ioutil"
+)
+
+/**
+* Copy file from source to destination
+* @method copyFile
+* @param   {os.FileInfo}	fi	file descriptors / info
+* @param   {String}    		src     source file
+* @param   {String}    		dest    destination file
+* @return  {nil / error}		success - return nil; error - return error description
+*/
+func copyFile(fi os.FileInfo, src, dest string) error {
```

Comment:
i would probably leverage existing pkg like `https://godoc.org/github.com/otiai10/copy` which also catches other cases like symlink, and this can simplify the test for this go code.

Reply:
and you need to have a test suite for this too

Reply:
thought of, but it's same and with this later we can control logics on copy using buffers; yes working on tests

Reply:
updated to https://godoc.org/github.com/otiai10/copy; will change later if needed